windows/helpers/phase1/install_7zip.ps1

```diff
@@ -18,18 +19,17 @@ if($isInstalled -and $isCurrent){
 # so not installed is the same as not current
 
 # Script directory is $PSScriptRoot
-$shortenedver = $Version.Replace('.','')
-$sevenzip="https://www.7-zip.org/a/7z$($shortenedver)-x64.exe"
+$sevenzip="https://github.com/mcmilk/7-Zip-zstd/releases/download/v$($Version)-v$($ZstdVersion)-R2/7z$($Version)-zstd-x64.exe"
 
 Write-Host -ForegroundColor Green "Installing 7zip $sevenzip"
 $out = "$($PSScriptRoot)\7zip.exe"
 Get-RemoteFile -RemoteFile $sevenzip -LocalFile $out -VerifyHash $Sha256
 
 Start-Process $out -ArgumentList '/S' -Wait
 Remove-Item $out
-Add-ToPath -NewPath "c:\program files\7-zip" -Local -Global
+Add-ToPath -NewPath "c:\program files\7-zip-Zstandard" -Local -Global
 
 ## Write the version key out to the registry
-Set-InstalledVersionKey -Component "sevenzip" -Keyname "Version" -TargetValue $Version
+Set-InstalledVersionKey -Component "sevenzip" -Keyname "version" -TargetValue $Version
```

Comment:
is it possible for `$ZstdVersion` to change but `$Version` to remain the same? if so we should combine them for the `Get-InstalledUpgradeStatus` and `Set-InstalledVersionKey` commands.

Reply:
Yes I think it's possible, let me update as you said